routes/users/getUserInfo.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const userController = require("../../controllers/users/users");
+const router = express.Router();
+
+router.get("/get-user-info", userController.getUserInfo);
```

Comment:
Use the hyphen (-) character to improve the readability of names in **ONLY** long path segments